java/hsfs/src/main/java/com/logicalclocks/hsfs/FeatureView.java

```diff
@@ -0,0 +1,27 @@
+/*
+ *  Copyright (c) 2025. Hopsworks AB
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ *  See the License for the specific language governing permissions and limitations under the License.
+ *
+ */
+
+package com.logicalclocks.hsfs;
+
+import com.logicalclocks.hsfs.constructor.Query;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@NoArgsConstructor
+public class FeatureView extends FeatureViewBase<FeatureView, FeatureStore, Query, List<Object>> {
```

Comment:
I am not entirely sure if we even need Base classes any more, since you can always just extend the class from hsfs without having it.

Also there is some discrepancy with the role of base class. In some places it defines only abstract method in others the implementation (even though in both cases work is only with metadata).
for example `public StreamFeatureGroup getStreamFeatureGroup(` does the same stuff in all engines.

Reply:
Each compute framework has its own StreamFeatureGroup that has its own FeatureGroupEngine. They are not the same StreamFeatureGroup(s).

Reply:
OK i think this could be improved, but not necessary to be done right now.